flocker/node/agents/cinder.py

```diff
@@ -331,6 +347,7 @@ def compute_instance_id(self):
         """
         local_ips = get_all_ips()
         api_ip_map = {}
+        return u'cf3824fb-0342-4e3f-9dc6-b5d0bb4a97c3'
```

Comment:
Sorry, should have added a note to ignore this change. Couldn't reproduce/test over limit issue (due to FLOC-2250) without hardcoding my instance.